package.json

```diff
@@ -0,0 +1,60 @@
+{
+    "name": "ilib-loctool-webos-c",
+    "version": "1.0.0",
+    "main": "./CFileType.js",
+    "description": "A loctool plugin that knows how to process C files",
+    "license": "Apache-2.0",
+    "keywords": [
+        "internationalization",
+        "i18n",
+        "localization",
+        "l10n",
+        "globalization",
+        "g11n",
+        "strings",
+        "resources",
+        "locale",
+        "translation",
+        "webpack",
+        "loader",
+        "javascript"
```

Comment:
change the keyword to "c" instead of "javascript". Also, we don't need "loader" or "webpack"

Reply:
I updated it.